automation_tools/satellite6/upgrade/capsule.py

```diff
@@ -26,23 +26,16 @@ def satellite6_capsule_setup(sat_host, os_version):
     # For User Defined Capsule
     if os.environ.get('CAPSULE_HOSTNAMES'):
         cap_hosts = os.environ.get('CAPSULE_HOSTNAMES')
-        cap_subscription = os.environ.get('CAPSULE_SUBSCRIPTION')
-        if not cap_subscription:
-            print('CAPSULE_SUBSCRIPTION environment variable is not '
-                  'defined !')
+        if not os.environ.get('CAPSULE_AK'):
+            print('CAPSULE_AK environment variable is not defined !')
             sys.exit(1)
-        elif len(cap_subscription.split(',')) != 3:
```

Comment:
Once you are exiting on line above, no need to elif condition. Same reason as my other comment about handling exceptions on begining ;)